### DISCUSSION TITLE
How to add custom picklist fields to picklist/relations

### DISCUSSION THREAD

**Discussion by PercyP** [YetiForceCompany/YetiForceCRM#15282] [How to] [answered]
Hi, 

I have created a couple of custom picklist fields in the Leads module and trying to create some picklist relations. However, the new fields don't show up as a selectable option in the Fields/Picklist/Relations area for leads. 

Is there some setting to get these to display as options?

Kind regards

**Comment by mariuszkrzaczkowski** [comment]
add screenshots for better understanding

**Comment by PercyP** [comment, marked as answer]
@mariuszkrzaczkowski 

I figured out what i was doing wrong I was using a multi-select list. 

Now works :)
